src/rooms/rooms.controller.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+  Query,
+} from '@nestjs/common';
+import { RoomsService } from './rooms.service';
+import { CreateRoomDto } from './dto/create-room.dto';
+import { UpdateRoomDto } from './dto/update-room.dto';
+import { ShowRoomDto } from './dto/show-room.dto';
+import { plainToClass } from 'class-transformer';
+
+@Controller('rooms')
+export class RoomsController {
+  constructor(private readonly roomsService: RoomsService) {}
+
+  @Post()
+  async createRoom(@Body() createRoomDto: CreateRoomDto) {
+    return this.roomsService.createRoom();
```

Comment:
```suggestion
    return this.roomsService.createRoom(createRoomDto);
```

여기서 createRoomDto 전달해줘야 하지 않나용?